ServerCore/Pages/Teams/Answers.cshtml

```diff
@@ -22,7 +22,7 @@
             </th>
             <th>
                 <a asp-page="./Answers" asp-route-teamId="@Model.TeamID" asp-route-sort="@(Model.SortForColumnLink(AnswersModel.SortOrder.GroupAscending, AnswersModel.SortOrder.GroupDescending))">
-                    Group
+                    @(Model.Event.TermForGroup ?? "Group")
```

Comment:
While you're updating stuff like this, does https://github.com/PuzzleServer/mainpuzzleserver/blob/main/ServerCore/Pages/Puzzles/Delete.cshtml#L67 need this update as well?

Reply:
I decided to leave the CRUD pages unchanged to make absolutely sure that editing the raw HTML syntax worked correctly.